pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -479,6 +484,120 @@ def get_hamiltonian(self, time: float) -> qutip.Qobj:
             time / 1000
         )  # Creates new Qutip.Qobj
 
+    @staticmethod
+    def get_min_variation(ch_sample: ChannelSamples) -> int:
```

Comment:
I think this should stay internal, so I'd rename it to 
```suggestion
    def _get_min_variation(ch_sample: ChannelSamples) -> int:
```
Also, out of context it's hard to figure out what it is doing (and why) at a glance so a small docstring would help